media-tv/kodi/files/kodi-17.3-ftpparse_string.patch

```diff
@@ -0,0 +1,82 @@
+From 634153c85115b44015cdb1ddd6263f5a87e4e41b Mon Sep 17 00:00:00 2001
```

Comment:
Please put the bug number in summary line instead of this useless `Gentoo-bug` so that it's visible in `git log --oneline`.